flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/ResultPartitionTest.java

```diff
@@ -217,6 +218,24 @@ public void testAddOnBlockingPartition() throws Exception {
 		testAddOnPartition(ResultPartitionType.BLOCKING);
 	}
 
+	/**
+	 * Tests {@link ResultPartition#createSubpartitionView(int, BufferAvailabilityListener)} would throw
```

Comment:
this is incorrect. `ResultPartition#createSubpartitionView` is not throwing the `PartitionNotFoundException`, `ResultPartitionManager#createSubpartitionView` does since the partition is no longer registered.

Reply:
Yes, my initial thought was when the registered partition was released to remove from `ResultPartitionManager`, then it would also throw `PartitionNotFoundException` during creating view in manger. I should adjust the comment properly.